src/validation.cpp

```diff
@@ -5078,7 +5080,11 @@ bool LoadMempool(CTxMemPool& pool)
         file >> mapDeltas;
 
         for (const auto& i : mapDeltas) {
-            pool.PrioritiseTransaction(i.first, i.second);
+            if (!MoneyRange(i.second)) {
+                return false;
+            } else {
```

Comment:
```suggestion
```

nit: No need to change indentation on early-return. (same above)

Reply:
Sorry, I didn't get it. Can you explain a bit what are you suggesting here?

Reply:
This was fixed in your last push AFAICT :)